### PR TITLE
New version: Google.Chrome version 106.0.5249.62

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | Google Chrome |     |
| Version     | 106.0.5249.62     |  |
| Publisher   | Google LLC   |       |
| ProductCode |           |     |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80885)

### CODE REVIEW

**Comment by yao-msft** [comment]
The installer url is pointing to msi now but all the metadata in the manifest are still for previous exe. For example, InstallerType and custom InstallerSwitch

File: manifests/g/Google/Chrome/106.0.5249.62/Google.Chrome.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Created with YamlCreate.ps1 v2.1.3 using InputObject 🤖 $debug=QUSU.7-2-6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Google.Chrome
+PackageVersion: 106.0.5249.62
+MinimumOSVersion: 10.0.0
```
Comment (yao-msft) [inline manifests/g/Google/Chrome/106.0.5249.62/Google.Chrome.installer.yaml:6]: MinOSVersion should be 10.0.0.0

**Comment by yao-msft** [comment]
Unattended installation failed probably because of old custom switches.
image

**Review by jedieaston — APPROVED** [review]

**Pull request merged**